README.md

```diff
@@ -113,7 +160,7 @@ Your Answer:
 
 Select all that apply:
 ```
-[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
+[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing. // what does the question mean by "pace"? seems too general a term to be marked as true.  "pace" usually depends on if the program is multi-threaded.
 [] Synchronous code runs all at the same time, asynchronous code runs completely randomly
-[] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
+[x] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
```

Comment:
:+1:  good question above! pace literally means speed here, and you're right that it is too general to be correct :)